editor/editor_interface.h

```diff
@@ -171,6 +171,7 @@ class EditorInterface : public Object {
 	void reload_scene_from_path(const String &scene_path);
 
 	PackedStringArray get_open_scenes() const;
+	TypedArray<Node> get_open_scenes_roots() const;
```

Comment:
```suggestion
	TypedArray<Node> get_open_scene_roots() const;
```
The noun here is singular in English (think "dog catchers")